analyzer/src/main/kotlin/PackageManager.kt

```diff
@@ -19,6 +19,7 @@
 
 package org.ossreviewtoolkit.analyzer
 
+import java.io.Closeable
```

Comment:
Commit message: I believe saying "properly close PackageManager instances" is misleading, as before this change package managers were not closeable, so you also couldn't close them.

Reply:
ok, changed.